test/TestAssets/TestProcess32/Program.cs

```diff
@@ -0,0 +1,4 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+Console.WriteLine($"Am I 32bit?: {!Environment.Is64BitProcess}");
```

Comment:
@MarcoRossignoli I haven't added any check here as I am now forcing the environment variable from calling point so it seemed to be adding no value. Let me know if you think we should still make some checks here.

Reply:
I would add the same...it's true that we're forcing...but it could work also for "fallback" reason...if for some reason we'll update the test. We should check that we're resolving correctly checking env var and the location of `typeof(object).Assembly.Location` directory